api/src/main/resources/MetadataTermMapping.hbm.xml

```diff
@@ -20,7 +20,7 @@ Copyright (C) OpenMRS, LLC.  All Rights Reserved.
 		   batch-size="25">
 		<id name="metadataTermMappingId" type="java.lang.Integer" column="metadata_term_mapping_id">
 			<generator class="native">
-				<param name="sequence">metadatamapping_metadata_term_mapping_metadata_term_mapping_id_seq</param>
```

Comment:
Isn't the original the correct one?

Reply:
That's what I thought when I added the sequence name in the beginning but after cross checking with the name of sequence created in db I found it to be **metadatamapping_metadata_term_mapp_metadata_term_mapping_id_seq** . Here is the link to image : https://pasteboard.co/JiZfhuB.png

Reply:
Shouldn't we just fix the wrong name that you found?

Reply:
A change set to rename the sequence in db for postgresql can be added rather then changing in hibernate mapping. Will this be right ?

Reply:
Anything that does not mean using the wrong name.

Reply:
@dkayiwa , there is a limit to maximum length of the name of sequence. I tried to rename the sequence with query : 

`ALTER SEQUENCE metadatamapping_metadata_term_mapp_metadata_term_mapping_id_seq RENAME TO metadatamapping_metadata_term_mapping_metadata_term_mapping_id_seq;`

But then we get output as :
```
NOTICE:  identifier "metadatamapping_metadata_term_mapping_metadata_term_mapping_id_seq" will be truncated to "metadatamapping_metadata_term_mapping_metadata_term_mapping_id_"
ALTER SEQUENCE

Query returned successfully in 94 msec.
```

So postgresql can never store the original sequence name we need.

Reply:
oh i see!

Reply:
I will add this in ticket description.